### ISSUE TITLE
the file `.env` is not included in the puppetlabs-kubernetes module on the Forge

### ISSUE THREAD

**Issue by tskirvin** [puppetlabs/puppetlabs-kubernetes#126]
## What you expected to happen?

The documentation in the README.md file indicates that there is a `.env` file included with the package that you can use to build hiera data for deploying Kubernetes via puppet.  This file is not actually included in the puppetlabs module (e.g. https://forge.puppet.com/v3/files/puppetlabs-kubernetes-2.0.2.tar.gz).  It is in the github page.

## How to reproduce it?

```
wget https://forge.puppet.com/v3/files/puppetlabs-kubernetes-2.0.2.tar.gz
tar -xzf puppetlabs-kubernetes-2.0.2.tar.gz
ls -la puppetlabs-kubernetes-2.0.2
```

**Comment by davejrt** [comment]
We'll make sure it's included in the next release

**Comment by davejrt** [comment]
Fixed as of release 3.0.0
